packages/salesforcedx-vscode-apex-replay-debugger/README.md

```diff
@@ -1,14 +1,109 @@
+    * Executing manual steps in your org in a web browser 
+1. To get a list of debug logs in your org, run **SFDX: Get Apex Debug Logs**.
+1. Click the log that you want to replay. The log downloads and opens in VS Code.
+1. Run **SFDX: Launch Apex Replay Debugger with Current File**.
+
+### Set Up an Apex Replay Debugger Session for a Sandbox or Production Org
+
+If you’re not using a scratch org, download a debug log from your org before you start debugging. Open that log in VS Code and then start a debugging session.
+
+1. In VS Code, open the debug log that you want to analyze.
+1. Run **SFDX: Launch Apex Replay Debugger with Current File**.  
+
+TIP: If your log file is part of your Salesforce DX project, you don’t need to open the log file and then run a separate command. Instead, you can find a log file in the Explorer view, right-click it, and then select **Launch Apex Replay Debugger with Current File**.
 
 ## Debug Your Code
 
+Set breakpoints, then replay your debug log and inspect your variables’ values.
+
+1. To set line breakpoints, open a `.cls` or `.trigger` file and click the column to the left of the line numbers.
+1. To switch to VS Code’s Debug view, click the bug icon on the left edge of the window.
+1. To replay the code execution that was logged in your debug log until you hit your first breakpoint, click the green play icon in the Debug actions pane at the top of the editor.
+1. Step through your code and examine the states of your variables in the VARIABLES section of the Debug view. For details, see [Debugging](https://code.visualstudio.com/docs/editor/debugging) in the Visual Studio Code docs.
+1. When you’ve stepped through all of the logged events, the debugging session ends. If you want to start again at the beginning of the log, run **SFDX: Launch Apex Replay Debugger with Last Log File**.
+
+## Considerations
+Keep these limitations and known issues in mind when working with Apex Replay Debugger.
+
+* You can replay only one debug log at a time. This can make it difficult to debug asynchronous Apex, which produces multiple debug logs.
+* You can’t replay a debug log generated by scheduled Apex.
+* Long string variable values are truncated.
+* Viewing a standard or custom object, you can drill down only to its immediate child variables (one level deep).
+* You can’t expand a collection (a list, set, or map), because its members are shown in their string form.
+* Modifying a collection does not update the collection variable in Variables view.
+* You can’t set method or conditional breakpoints.
+* You can’t evaluate or watch variables or expressions in the Debug view’s WATCH section.
+* While debugging, right-clicking a variable in the VARIABLES section of the Debug view and selecting **Copy Value** works properly, but **Copy as Expression** and **Add to Watch** don’t work as expected. 
```

Comment:
"...but **Copy as Expression** and **Add to Watch** don’t work as expected. ".  I'm having a hard time parsing to see what the "don't work as expected" mean.

![screen shot 2018-06-06 at 4 52 57 pm](https://user-images.githubusercontent.com/54414/41070899-18b3cade-69aa-11e8-9038-812c29ebf098.png)

Does it mean that "they do not work like they were intended to" or "they do not work, and this is expected". The latter is true but I'm worried that the phrase here might imply the former.

Reply:
Actually, disregard. What you have so far is fine. I understand the intent now.